### PR TITLE
Add flask sample

### PR DESCRIPTION
**Pull request by kouul**

### CODE REVIEW

**Comment by glours** [comment]
Hello @kouul,

Thanks for your PR! 
We already have 2 samples with flask, [one with nginx and mongo](https://github.com/docker/awesome-compose/tree/master/nginx-flask-mongo) and [another one with nginx and mysql](https://github.com/docker/awesome-compose/tree/master/nginx-flask-mysql)
Maybe we just need to improve the existing ones if needed, what do you think?

**Comment by kouul** [comment]
Hi @glours 

Yes, but there is no sample of a plain flask application. By plain I mean without any connection to any db. It could be easier for someone to use this one to connect to another db other than Mysql or mongo.

**Comment by aiordache** [comment]
Thank you @kouul 
It sounds good to me to have more single service samples, we added some already. If I would like to run a simple python service, this may be useful as I wouldn't need to cleanup the other samples just to get my flask one.

File: flask/app/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM python:3.7-alpine 
+WORKDIR /app 
+COPY . /app 
+RUN pip3 install -r requirements.txt
```
Comment (glours) [inline flask/app/Dockerfile]: Can you start with the installation of requirements before copying the source code to take advantage of layers caching

File: flask/app/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+FROM python:3.7-alpine 
+WORKDIR /app 
+COPY . /app 
+RUN pip3 install -r requirements.txt 
+ENTRYPOINT ["python3"] 
+CMD ["app.py"]
```
Comment (glours) [inline flask/app/Dockerfile]: nit: add a new line at the end of the file

File: flask/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+version: '3' 
+services: 
+  web: 
+    build: app 
+    ports: 
+      - '5000:5000'
```
Comment (glours) [inline flask/docker-compose.yml]: nit: add a new line at the end of the file

File: flask/app/Dockerfile (hunk 1 above)
Comment (kouul) [reply flask/app/Dockerfile]: Done

File: flask/app/Dockerfile (hunk 2 above)
Comment (kouul) [reply flask/app/Dockerfile]: Done

File: flask/docker-compose.yml (hunk 3 above)
Comment (kouul) [reply flask/docker-compose.yml]: Done

**Pull request merged**